repository/src/main/resources/db/changelog/db.changelog-rdm-9824.xml

```diff
@@ -0,0 +1,13 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-9824" author="kiran.yenigala@hmcts.net">
+
+        <addUniqueConstraint
+                columnNames="id, case_type_id, display_order, question_id"
```

Comment:
this constraint would allow 2 questions with same `case_type_id`, `id`, `question_id` if the `display_order` is different. Does it make sense to allow that?

Reply:
after discussion with Mario, we are keeping this. Handled this on Java side